tasks/instrument-test-resources.js

```diff
@@ -27,24 +27,28 @@ async function main() {
     './packages/jest-runner/testResources/jasmine2-node/src/Add.js': './packages/jest-runner/testResources/jasmine2-node-instrumented/src/Add.js',
     './packages/jest-runner/testResources/jasmine2-node/src/Circle.js': './packages/jest-runner/testResources/jasmine2-node-instrumented/src/Circle.js',
   },  '__stryker2__')
+  await instrument({
```

Comment:
We might be able to change an existing scenario, right? Instead of adding one 🤷‍♂️ Or is there a specific reason for another one?

Reply:
I initially added it to `jasmine2-node` but then started to break other tests and then I found the bailing problem and wanted to isolate the test.

I'll give it a go. Should be simple enough

Reply:
It might be better to add it to the jest-circus integration tests since jest-circus is now the default for jest since v27